charts/chart.area.js

```diff
@@ -133,6 +133,10 @@ area.prototype.getSpec = function() {
 
 
 function getAreaMark(config, metadata){
+    interpolation = "linear";
```

Comment:
@chamilad Can you add this to defaults in the utils [1]. Then you can avoid hard coding it as linear and checking with if conditions
[1] https://github.com/wso2/VizGrammar/blob/master/charts/chart.util.js#L32